### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by feifeibear**
fix bugs in the BERT cpp output return value.
fix bugs in the BERT classification example. 
Add QBert with onnxruntime int8.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/TurboTransformers?pullRequest=154) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/TurboTransformers?pullRequest=154) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: flans39<br/>:white_check_mark: xcnick<br/>:white_check_mark: feifeibear<br/>:x: shicheng0829<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/TurboTransformers?pullRequest=154) it.</sub>

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/TurboTransformers?pullRequest=154) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/TurboTransformers?pullRequest=154) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: xcnick<br/>:white_check_mark: flans39<br/>:white_check_mark: feifeibear<br/>:x: shicheng0829<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/TurboTransformers?pullRequest=154) it.</sub>

File: turbo_transformers/python/tests/bert_model_test.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def check_torch_and_turbo(self, use_cuda):
 
         self.assertTrue(
             numpy.allclose(torch_result[0].cpu(),
-                           turbo_result[0].cpu(),
+                           turbo_result[0],
```
Comment (xcnick) [inline turbo_transformers/python/tests/bert_model_test.py:68]: 这个地方应该要加上.cpu()吧？

Comment (feifeibear) [reply turbo_transformers/python/tests/bert_model_test.py:68]: 嗯，要加！下个PR加上就行

**Review by fangjiarui — APPROVED** [review]

**Pull request merged**